Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.py

```diff
@@ -1502,57 +1502,49 @@ def get_impacted_resources(client: PrismaCloudComputeClient, args: dict) -> Comm
     limit, offset = parse_limit_and_offset_values(limit=args.pop("limit", "50"), offset=args.pop("offset", "0"))
     cves = argToList(arg=args.get("cve", []))
 
-    impacted_images, impacted_hosts, context_output = [], [], []
+    context_output, raw_response = [], []
+    final_impacted_resources = {"images": [], "registryImages": [], "hosts": [], "functions": []}
     for cve in cves:
         # api does not support offset/limit
         if cve_impacted_resources := client.get_impacted_resources(cve=cve):
-            if "riskTree" in cve_impacted_resources and cve_impacted_resources.get("riskTree") is not None:
-                cve_impacted_resources["riskTree"] = dict(
-                    filter_api_response(
-                        api_response=list(cve_impacted_resources.get("riskTree", {}).items()),  # type: ignore[arg-type]
+            raw_response.append(cve_impacted_resources)
```

Comment:
lets make the `raw_response` a `dict` instead of a `list` where the keys are the CVEs and their values is the raw-response for each CVE.

example:
```
raw_responses = {'CVE1': raw_response_1, 'CVE2;, raw_response_2}
```

Would be easier to understand what CVE belongs to which raw response as we do api requests as the number of the CVEs